src/Generator/BilligerDE.php

```diff
@@ -326,7 +300,11 @@ private function buildRow($variation, KeyValue $settings, $attributes)
                 'features'      => $this->elasticExportPropertyHelper->getProperty($variation, 'features', self::BILLIGER_DE, $settings->get('lang')),
                 'style'         => $this->elasticExportPropertyHelper->getProperty($variation, 'style', self::BILLIGER_DE, $settings->get('lang')),
                 'old_price'     => $priceList['oldPrice'],
-                'images'        => $this->getAdditionalImages($imageList),
+                'images'        => $imageList,
+
+                // needed for SOP(Solute Order Platform) market
+                'delivery_sop'      => $this->elasticExportPropertyHelper->getProperty($variation, 'delivery_sop', self::BILLIGER_DE, $settings->get('lang')),
```

Comment:
pls make delivery_sop a constant. 
also just check if it was set. it does not need to have a value